ext/curl/interface.c

```diff
@@ -2557,6 +2557,11 @@ PHP_FUNCTION(curl_getinfo)
 		if (curl_easy_getinfo(ch->cp, CURLINFO_STARTTRANSFER_TIME_T, &co) == CURLE_OK) {
 			CAAL("starttransfer_time_us", co);
 		}
+#if LIBCURL_VERSION_NUM >= 0x080a00 /* Available since 8.10.0 */
+		if (curl_easy_getinfo(ch->cp, CURLINFO_POSTTRANSFER_TIME_T, &s_code) == CURLE_OK) {
```

Comment:
question: are you sure s_code should be used here ?

Reply:
You are right it indeed should be a `curl_off_t`, so using `co` instead of `s_code` now. Thank you.